examples/website/src/shared/blocks/peerPageBreak.tsx

```diff
@@ -78,6 +93,15 @@ export function PageBreakBlock({
             <Image src={peer.logoURL} height={90} width={90} />
           </div>
         )}
+        <pre>
```

Comment:
That's not really useful rather just leave it as it was

Reply:
updated